src/VisualStudio/Core/Def/Implementation/Options/RoamingVisualStudioProfileOptionPersister.cs

```diff
@@ -132,9 +132,16 @@ public bool TryFetch(OptionKey optionKey, out object value)
                 }
                 else
                 {
-                    value = optionKey.Option.DefaultValue;
+                    value = null;
+                    return false;
```

Comment:
Agreed. I'll clean that up in a separate PR since that'll be a bit more invasive.